README.md

```diff
@@ -8,15 +13,26 @@ therefore entries (de)serialization in front of the cache will be needed in most
 
 ### Simple initialization
 
-```go
-import "github.com/allegro/bigcache"
+```
+package main
+
+import (
+        "github.com/hiqinternational/bigcache"
+        "time"
+        "fmt"
+)
+
+
+func main(){
+        cache, _ := bigcache.NewBigCache(bigcache.DefaultConfig(10 * time.Minute))
 
-cache, _ := bigcache.NewBigCache(bigcache.DefaultConfig(10 * time.Minute))
 
-cache.Set("my-unique-key", []byte("value"))
+        fmt.Printf("Initialization of cache is slow... normal.\n")
```

Comment:
?

Reply:
Please remove this statement, the cache initialization is not slow.